mateo-mustapic/assignments/liferay-app/sdk/portlets/todo-portlet/docroot/js/main.js

```diff
@@ -20,7 +28,45 @@ AUI.add(
 					initializer: function(config) {
 						var instance = this;
 
-						var todoList = instance.one('.task-container ul');
+						new A.CharCounter(
+							{
+								counter: '.counter',
+								input: '.input-item',
+								maxLength: 35
+							});
+
+						var todoList = A.one('.task-container ul');
+
+						var finishedTask = A.one('.finished-task');
+
+						var remainingDiv = A.one('.remaining-tasks-count');
+
+						var profileTasksCount = A.one('.profile-tasks-count');
```

Comment:
Please make the change previously requested in https://github.com/markocikos/liferay-internship/pull/149#discussion_r124256379:

Use ```A.one()``` only if there is no other way, for example is some component adds ```<div>``` in root HTML. Typically you want to query either by ID (```instance.byId()```) or by class relative to container (```node.one()```).

```A.one()``` results in bad performance and inability to have instancable portlets.